plugins/modules/azure_rm_postgresqlserver.py

```diff
@@ -51,6 +51,15 @@
         description:
             - The maximum storage allowed for a server.
         type: int
+    geo_redundant_backup:
+        description:
+            - Choose between locally redundant(default) or geo-redundant backup. This cannot be updated after first deployment
+        type: bool
+        default: default
```

Comment:
Argument 'geo_redundant_backup' in documentation defines default as ('default') but this is incompatible with parameter type 'bool'

Reply:
@karisN Please keep consistent with the argument spec defined. Thank you very much!